frontend/src/app/components/MainInstallButton.js

```diff
@@ -41,10 +41,10 @@ export default class MainInstallButton extends React.Component {
     const { isFirefox, isMinFirefox, isMobile, hasAddon } = this.props;
     const isInstalling = this.state.isInstalling && !hasAddon;
     const terms = <Localized id="landingLegalNoticeTermsOfUse">
-      <a href="/terms">Terms of Use</a>
+      <a href="/terms"/>
```

Comment:
While trying to debug this issue, i noticed that these links were not getting l10n-ized as well. Removing strings from src files seems to resolve this issue

Reply:
We need to have default strings in these elements going forward, though, per #2791. If having a string in there is causing a problem, then that might be an issue with localization that we need to fix.

Reply:
@lmorchard AFAIK these are the only two examples where we're passing in links like this. I'm not super familiar with fluent react, but we can ref this PR on that bug maybe?

Reply:
Should probably file a new issue, since #2791 is closed

Reply:
Looks like #2911 is filed now. Will merge this and see if we can work things out to get the original text put back in